R/cov_search.R

```diff
@@ -13,6 +13,16 @@
 #'
 MLCovSearch <- function(tab, list_pop_param, cov_continuous, cov_factors, seed = 123) {
   
+  # Check that covariates supplied by user exist in the data
+  errors <- data_validation(tab, list_pop_param, cov_continuous, cov_factors)
+  if (length(errors) > 0) {
+    cat("Stopping due to errors detected in data validation:\n")
+    cat(errors, sep = "\n")
+    return(invisible())
+  } else {
+    cat("Data validation successful, selecting covariates...\n")
```

Comment:
this condition can be removed - no need tell the user anything if everything is OK